File: Python/EX26.py
numero = int(input('digite um numero positivo: '))

while(numero<=0):
    print("Digite somente numeros positivos")
    numero = int(input('digite um numero positivo: '))

print("Tabuada do ",numero)

for i in range( 1, 11 , 1)	:
	
	tab = numero*i
	print(numero," x",i," = " , tab)

File: C/EX26.cpp
#include <conio.h>
#include <stdlib.h>
#include <stdio.h>
#include <time.h> 
#include <string.h>
#include <math.h>
#include <locale.h>
#include <ctype.h>
#include <iostream>


int x,num,tab;

int main()
{
	while(1)
	{
	
	ab:
	printf("Insira um numero positivo para saber a tabuada\n");
	scanf("%d",&num);
	if(num<0)
	{
		printf("Insira somente numeros positivos\n");
		goto ab;
	}
	printf("Tabuada do %d\n",num);
	for(x=0 ; x<=10 ; x++)
	{
		tab = num*x;
		printf("%dx%d = %d \n", num,x,tab);
	}
}
}
